Dockerfile.local

```diff
@@ -1,79 +1,87 @@
-# add the -race parameter to go build call in order to instrument with race condition detector: https://blog.golang.org/race-detector
+RUN : \
+    && go version \
+    && go test ./...
 
+# Build plugins and threagile
+RUN : \
+    && go build -a -trimpath -ldflags="-s -w -X main.buildTimestamp=$(date '+%Y%m%d%H%M%S')" -buildmode=plugin -o raa.so raa/raa/raa.go \
+    && go build -a -trimpath -ldflags="-s -w -X main.buildTimestamp=$(date '+%Y%m%d%H%M%S')" -buildmode=plugin -o dummy.so raa/dummy/dummy.go \
+    && go build -a -trimpath -ldflags="-s -w -X main.buildTimestamp=$(date '+%Y%m%d%H%M%S')" -buildmode=plugin -o demo-rule.so risks/custom/demo/demo-rule.go \
+    && go build -a -trimpath -ldflags="-s -w -X main.buildTimestamp=$(date '+%Y%m%d%H%M%S')" -o threagile
 
+# Copy files in build image to simplify copy instruction
+RUN : \
+    && cp /app/demo/example/threagile.yaml /app/demo/example/threagile-example-model.yaml \
+    && cp /app/demo/stub/threagile.yaml /app/demo/stub/threagile-stub-model.yaml
 
 
 ######
-## Stage 3: Make final small image
+## Stage 2: Make final small image
 ######
-FROM alpine
+FROM docker.io/library/alpine:latest
 
-# label used in other scripts to filter
+# Label used in other scripts to filter
 LABEL type="threagile"
 
-# add certificates
-RUN apk add ca-certificates
-# add graphviz, fonts
-RUN apk add --update --no-cache graphviz ttf-freefont
-# https://stackoverflow.com/questions/66963068/docker-alpine-executable-binary-not-found-even-if-in-path
-RUN apk add libc6-compat
-# https://stackoverflow.com/questions/34729748/installed-go-binary-not-found-in-path-on-alpine-linux-docker
-# RUN mkdir -p /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
-# clean apk cache
-RUN rm -rf /var/cache/apk/*
+RUN apk add --no-cache \
+    ca-certificates \
+    graphviz \
+    ttf-freefont
 
 WORKDIR /app
 
-COPY --from=build /app/threagile /app/threagile
-COPY --from=build /app/raa.so /app/raa.so
-COPY --from=build /app/dummy.so /app/dummy.so
-COPY --from=build /app/demo-rule.so /app/demo-rule.so
-COPY --from=build /app/LICENSE.txt /app/LICENSE.txt
-COPY --from=build /app/report/template/background.pdf /app/background.pdf
-COPY --from=build /app/support/openapi.yaml /app/openapi.yaml
-COPY --from=build /app/support/schema.json /app/schema.json
-COPY --from=build /app/support/live-templates.txt /app/live-templates.txt
-COPY --from=build /app/support/render-data-asset-diagram.sh /app/render-data-asset-diagram.sh
-COPY --from=build /app/support/render-data-flow-diagram.sh /app/render-data-flow-diagram.sh
-COPY --from=build /app/server /app/server
-COPY --from=build /app/demo/example/threagile.yaml /app/threagile-example-model.yaml
-COPY --from=build /app/demo/stub/threagile.yaml /app/threagile-stub-model.yaml
-
-RUN mkdir /data
-
-RUN chown -R 1000:1000 /app /data
-USER 1000:1000
-
-ENV PATH=/app:$PATH
-ENV GIN_MODE=release
+# Add non-privileged user for running threagile
+RUN adduser \
```

Comment:
Kudos for adding best security practices for the project which is about improving security ;)